bin/metrics-docker-stats.rb

```diff
@@ -212,4 +220,17 @@ def blkio_stats(io_stats)
     end
     stats_out
   end
+
+  def calculate_cpu_percent(stats)
+    cpu_percent = 0.0
+    previous_cpu = stats['precpu_stats']['cpu_usage']['total_usage']
+    previous_system = stats['precpu_stats']['system_cpu_usage']
+    cpu_delta = stats['cpu_stats']['cpu_usage']['total_usage'] - previous_cpu
+    system_delta = stats['cpu_stats']['system_cpu_usage'] - previous_system
+    if system_delta > 0 && cpu_delta > 0
+      number_of_cpu = stats['cpu_stats']['cpu_usage']['percpu_usage'].length
+      cpu_percent = (cpu_delta.to_f / system_delta.to_f) * number_of_cpu * 100
```

Comment:
should we consider doing something like only displaying the first 2-4 decimal points? I doubt anyone cares the difference and will reduce the amount of storage in your metrics.